app/tests/daemon_script.py

```diff
@@ -0,0 +1,27 @@
+"""
+Test script for daemonizing
+"""
+
+from sys import argv
+from time import sleep
+
+from onedrive_client.utils import daemon
+
```

Comment:
I would like to see  a daemon script which is compatible with LSB specification http://refspecs.linuxbase.org/LSB_2.0.1/LSB-PDA/LSB-PDA/iniscrptact.html

ACTIONS = ['start', 'stop', 'restart', 'reload', 'status', 'try-restart', 'force-reload']